backend/tracim_backend/config.py

```diff
@@ -967,11 +968,15 @@ def _load_search_config(self):
         self.SEARCH__ELASTICSEARCH__INGEST__SIZE_LIMIT = int(
             self.get_raw_config("search.elasticsearch.ingest.size_limit", "52428800")
         )
-        self.SEARCH__ELASTICSEARCH__HOST = self.get_raw_config(
-            "search.elasticsearch.host", "localhost"
+        self.SEARCH__ELASTICSEARCH__HOST = string_to_list(
+            self.get_raw_config("search.elasticsearch.host", "localhost"),
+            cast_func=str,
+            separator=",",
         )
-        self.SEARCH__ELASTICSEARCH__PORT = int(
-            self.get_raw_config("search.elasticsearch.port", "9200")
+        self.SEARCH__ELASTICSEARCH__PORT = string_to_list(
+            self.get_raw_config("search.elasticsearch.port", "9200"),
+            cast_func=int,
+            separator=",",
```

Comment:
I suggest we make configuration more clear:

- have same number of parameters for host and port
- use only one value for port

`assert len(self.SEARCH__ELASTICSEARCH__HOST) == len(self.SEARCH__ELASTICSEARCH__PORT) or len(self.SEARCH__ELASTICSEARCH__PORT)==1`

A nice-to-have would be here to explicitly log the cluster members like:

```
elastic search cluster member: 192.168.1.258:9200
elastic search cluster member: 192.168.1.258:9201
elastic search cluster member: 192.168.1.259:9200
```

Reply:
asserts have been added, the nice-to-have has been skipped (no logging of this kind, we postpone it to a potential future development)